### ISSUE TITLE
new Slackhook() returns 'any' in Typescript

### ISSUE THREAD

**Issue by techieshark** [TheAppleFreak/winston-slack-webhook-transport#7]
Getting error when using the suggested code:

```
const slackLogger = winston.createLogger({
  level: "info", 
  transports: [
    new SlackHook({ 
      webhookUrl: "https://hooks.slack.com/services/xxx/xxx/xxx"
    })
  ]
});
```

> 'new' expression, whose target lacks a construct signature, implicitly has an 'any' type.ts(7009)

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/1072292/75314858-169e0100-58b5-11ea-81b2-89ac92a5ede4.png">

Probably due to "    "noImplicitAny": true, " in tsconfig.json.  But I think the types are wrong, no?

**Comment by TheAppleFreak** [comment]
Yeah, looks like I made a mistake while writing the types. I'll take a closer look at this over the next day or two and get that all patched up. Thank you for bringing this to my attention!

**Comment by techieshark** [comment]
Super, thanks!

**Comment by TheAppleFreak** [comment]
Alright, just published version 1.2.1 that has an updated typings file. Thank you once again for informing me of this!
